release-team/role-handbooks/docs/README.md

```diff
@@ -74,12 +106,102 @@ In addition to the time requirements above, a Docs Lead must:
+- [The krel `release-notes` subcommand](https://github.com/kubernetes/release/blob/master/docs/krel/release-notes.md)
+- [The old release notes tool](https://github.com/kubernetes/release/tree/master/cmd/release-notes)
+- [Release notes website](https://relnotes.k8s.io) *(note: release notes website is only required for the final release)*
+- [go-modiff](https://github.com/saschagrunert/go-modiff)
+- [Hackmd](https://hackmd.io/)
+- [LWKD](http://lwkd.info) *(note: consider contributing to LWKD as part of your role)*
+- [Kubernetes Documentation Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/)
+
+## Debugging Tips
+
+If you are having trouble running the `krel` tool, here are some common issues and solutions:
+
+1. Try running with `--log-level=debug` or `--log-level=trace` to get more information about what is going wrong.
+2. A temp directory gets created at `/var/folders/7t/273pt80d51l70mj4rxznq_lm0000gn/T/<k8s-hash>` when the `krel` tool is called.
+If this data is stale, you can try clearing to remove old data with `rm -rf /var/folders/7t/273pt80d51l70mj4rxznq_lm0000gn/T/k8s`
+
+Checkout the documentation for the [krel `release-notes` subcommand](https://github.com/kubernetes/release/blob/master/docs/krel/release-notes.md).
+
+## Release Notes File Structure
+
+All the release notes for a release are stored under the [releases](https://github.com/kubernetes/sig-release/tree/master/releases) 
+directory in the sig-release repo. 
+
+For each release there is a JSON and markdown file that contains the collected release notes across path releases. For example,
+the 1.30 release [markdown file](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.30/release-notes/release-notes-draft.md)
+contains all the correctly formatted release notes text for the 1.30 release. The [JSON file](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.30/release-notes/release-notes-draft.json)
+contains the release notes metadata that is used to generate the markdown file. 
+
+When a Docs team member runs the `krel release-notes` command, a new session is created so that you can pause and resume
+the editing process. For example the 1.30 release notes sessions are stored in the [sessions](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.30/release-notes/sessions)
+directory in the sig-release repo under `release-1.30`.
+
+If a Docs team member finds a mistake in the release notes, the edit will be saved as a map yaml file in the [maps](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.30/release-notes/maps)
+directory. These maps are used to generate the markdown file and JSON file with the correctly edited release note. 
+
+## TODOs
+
+As a Docs shadow, if you are interested in contributing to the improvement of the release notes process, consider the following
+areas of improvement:
+
+#### Github Workflow to Detect Common Release Note Issues
```

Comment:
@Vyom-Yadav is actively working on adding this! 😄 https://github.com/kubernetes-sigs/release-actions/pull/101 Once it goes in, we should remove this section and add a section on reading the errors reported from the GitHub workflow.

Reply:
@Vyom-Yadav Should this be left in for this release or can it be removed?

Reply:
Let's leave it for now. We can remove it once the workflow merges in sig-release